app/views/users/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<% set_page_title(title_with_error_prefix(@user.name, @user.errors.any?)) %>
+<% content_for :back_link, govuk_back_link_to(users_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <%= t("users.edit.title") %>
+    </h1>
+
+    <h2 class="govuk-heading-m"><%= t("users.edit.details") %></h2>
+    <%= govuk_summary_list(actions: false) do |summary_list|
+      summary_list.with_row do |row|
+        row.with_key { t('users.index.table_headings.name') }
+        row.with_value { @user.name }
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { t('users.index.table_headings.email') }
+        row.with_value { @user.email }
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { t('users.index.table_headings.organisation') }
+        row.with_value { @user.organisation_slug }
+      end
+    end %>
+
+    <%= form_with(model: @user, url: user_path) do |f| %>
+      <% if @user.errors.any? %>
+        <%= f.govuk_error_summary %>
+      <% end %>
+      <span class="govuk-caption-l"><%= @item_name %></span>
+      <%= f.govuk_collection_radio_buttons :role,       [
+        OpenStruct.new(label: t('users.roles.editor.name'), value: :editor, description: t('users.roles.editor.description')),
+        OpenStruct.new(label:t('users.roles.super_admin.name'), value: :super_admin, description: t('users.roles.super_admin.description')),
+      ], :value,:label, :description,
+      legend: { text: "Role", size: 'm', tag: 'h2' } %>
+
+    <%= f.govuk_submit t('users.save') do
+      govuk_link_to(t('users.cancel'), users_path, class: 'govuk-button govuk-button--secondary')
```

Comment:
Is it ok for the cancel button to just be a link?

Reply:
Can we use `govuk_button_link_to` instead?

https://govuk-components.netlify.app/helpers/button/#link-styled-like-a-button

Reply:
yes!